### ISSUE TITLE
Policy Templates - applicationSessionContext.sliceInfo.sd accepts invalid values

### ISSUE THREAD

**Issue by dsilhavy** [5G-MAG/rt-5gms-application-function#122]
## Description
The field `applicationSessionContext.sliceInfo.sd` seems to accept values of an invalid format. By definition in TS 26.512 17.7.0 and `TS29571_CommonData.yaml` `applicationSessionContext.sliceInfo.sd` needs to follow the following pattern: `^[A-Fa-f0-9]{6}$`. 

The following configuration is accepted when updating a dynamic policy configuration:

````json
    "applicationSessionContext": {
        "sliceInfo": {
            "sst": 200,
            "sd": "GGGG"
        }
    }
````

The `sd` field does not follow the aforementioned pattern and should be rejected.

Note that I am using the REST interface of the AF and not the m1-session-tool.

**Comment by dsilhavy** [comment]
Fixed in #125

**Comment by davidjwbbc** [comment]
Closed by merge of #125
